demos/Banner/main.vala

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg libadwaita-1
+
+public void main () {
+  var banner = workbench.builder.get_object ("banner") as Adw.Banner;
```

Comment:
Use C-Style casting instead:
`var banner = (Adw.Banner) workbench.builder.get_object ("banner");`

Reply:
I used the demo of "Toast" as reference, and it used the 'as' keyword, so I followed with that.